feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable } from '@angular/core';
+import { OrderEntry } from '@spartacus/core';
+import { LineItem } from './configurator-cart-entry-bundle-info.model';
+import { ConfigurationInfo } from '../../core/model/common-configurator.model';
+
+/**
+ * Service for mapping of the CPQ line items from order entry
+ */
+@Injectable({ providedIn: 'root' })
+export class ConfiguratorCartEntryBundleInfoService {
+  /**
+   * Retrieves the number of CPQ line items for an order entry
+   *
+   * @param {OrderEntry} entry - Order entry
+   * @returns {number} - number of line items
+   */
+  retrieveNumberOfLineItems(entry: OrderEntry): number {
```

Comment:
Is this function required? The number of lineItems is the `retrieveLineItems(entry).length`.

Reply:
Pick!

Reply:
Done!